fluentdhec/lambda_function.py

```diff
@@ -32,19 +34,41 @@ def lambda_handler(event, context):
         send_to_hec(payload)
 
 
+def extract_time(message):
+    res = False
+    regex = r'(?:[A-Z]+\s\[|usecs:)(?P<timestamp>[0-9\-\: ]+)'
+    match = re.search(regex, message)
+    if match:
+        res = match.group(1)
+        if not res.isdigit():
+            try:
+                res = datetime.strptime(res, '%Y-%m-%d %H:%M:%S').timestamp()
+            except Exception as e:
```

Comment:
Which exception does `datetime.strptime` throw when given a non matching time and format string? Can you just catch that?